RosNet/Exceptions/MissingConnectionException.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using RosNet.DataModel;
+
+namespace RosNet.Exceptions;
+
+/// <summary>
+/// Exception for when there are messages in the bag without corresponding connection
+/// </summary>
+/// <param name="message">Message which explains the cause</param>
+/// <param name="messages">List of the messages without the corresponding connection and the message data as bytearray</param>
+public class MissingConnectionException : Exception
```

Comment:
This is _never_ an issue with all other implementations of ROS-bags, since they read all the connections first, this having different failure-modes is **weird**.

Reply:
It is just invalid data either way, why should the user care about how the data is invalid?